### ISSUE TITLE
TypeError: Cannot read property 'content' of undefined

### ISSUE THREAD

**Issue by ex1st** [snapappointments/bootstrap-select#2436]
bootstrap: v3.4.1
bootstrap-select: v1.13.13

```javascript
TypeError: Cannot read property 'content' of undefined
    at Selectpicker.render (bootstrap-select.js:1722)
    at Selectpicker.refresh (bootstrap-select.js:2997)
    at HTMLSelectElement.<anonymous> (bootstrap-select.js:3125)
    at Function.each (jquery.js:367)
    at jQuery.fn.init.each (jquery.js:202)
    at jQuery.fn.init.Plugin [as selectpicker] (bootstrap-select.js:3097)
```

This error occurs on the render of empty "select" (without "option" elements). Thank you!

```javascript
if (option.title) {
  titleOptions.text = option.title;
} else if (thisData.content && that.options.showContent) { // <--- this
  titleOptions.content = thisData.content.toString();
  hasContent = true;
} else {
  if (that.options.showIcon) {
    titleOptions.icon = thisData.icon;
  }
  if (that.options.showSubtext && !that.multiple && thisData.subtext) titleOptions.subtext = ' ' + thisData.subtext;
  titleOptions.text = option.textContent.trim();
}
```

**Comment by caseyjhol** [comment]
Bug reports must include a live demo of the problem. Per our [contributing guidelines](https://github.com/snapappointments/bootstrap-select/blob/master/CONTRIBUTING.md), please create a reduced test case (you can use our preconfigured [Plunker](https://developer.snapappointments.com/bootstrap-select/playground/)), and report back with your link.

**Comment by ex1st** [comment]
@caseyjhol, done - https://plnkr.co/edit/xawicUkKv8ODeImJ

**Comment by caseyjhol** [comment]
Seems like it's kind of a fringe case (as there's no reason to call refresh before populating the select, and removing that first refresh() call fixes the bug). In any case, to cover all bases, this should be fixed.

**Comment by caseyjhol** [comment]
Released in [v1.13.14](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.14)!
